packages/template/vite/tmpl/vite.main.config.mjs

```diff
@@ -5,6 +5,7 @@ export default defineConfig({
   resolve: {
     // Some libs that can run in both Web and Node.js, such as `axios`, we need to tell Vite to build them in Node.js.
     browserField: false,
+    conditions: ['node'],
```

Comment:
```suggestion
    conditions: ['node-addons', 'node'],
```

Would we also want to add `"node-addons"` just in case a module exposes that?

Reply:
LGTM 👍